this patch is required to make coyz run, until the following commit gets merged upstream
https://github.com/geigi/cozy/pull/762

diff --git a/cozy/ui/widgets/whats_new_window.py b/cozy/ui/widgets/whats_new_window.py
index 1e80d1ef..1a79005b 100644
--- cozy/ui/widgets/whats_new_window.py
+++ cozy/ui/widgets/whats_new_window.py
@@ -51,12 +51,16 @@ def __init__(self, **kwargs):
     def _fill_window(self):
         self.children = []
 
-        last_launched_version = version.parse(self.app_settings.last_launched_version)
-
-        if type(last_launched_version) is version.LegacyVersion:
+        try:
+            last_launched_version = version.parse(self.app_settings.last_launched_version)
+        except version.InvalidVersion:
             self._fill_welcome()
         else:
-            self._fill_whats_new(last_launched_version)
+            if type(last_launched_version) is version.LegacyVersion:
+                self._fill_welcome()
+            else:
+                self._fill_whats_new(last_launched_version)
+
 
     def _fill_welcome(self):
         from cozy.ui.widgets.welcome import Welcome
